starmap_client/models.py

```diff
@@ -316,7 +323,7 @@ class BillingCodeRule(StarmapJSONDecodeMixin):
 
 
 @frozen
-class MappingResponseObject(StarmapJSONDecodeMixin, MetaMixin):
+class MappingResponseObject(MetaMixin, StarmapJSONDecodeMixin):
```

Comment:
NOTE: I had to swap the Mixins order since the `docs` step were failing with the other way around. It doesn't present any real changes in the implementation, though.